x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/http/sender/RequestExecutorService.java

```diff
@@ -251,69 +308,8 @@ public void send(HttpRequest request, @Nullable TimeValue timeout, ActionListene
         }
     }
 
-    /**
```

Comment:
Since we weren't using these anyway I decided to no longer extend the `ExecutorService` interface. I created the `RequestExecutor` interface instead.